test/functional/c/test_magnetic_dataset.cpp

```diff
@@ -362,4 +362,43 @@ TEST(MagneticDataset, test_failure_with_slightly_distorted_positions) {
     if (dataset) spg_free_magnetic_dataset(dataset);
 }
 
+TEST(MagneticDataset, test_with_right_handed_magnetic_lattice) {
+    double lattice[3][3] = {
```

Comment:
Can you add a comment here about the issue and that it covers an issue where the absolute value of determinant was taken?

Reply:
Done